content/zh/docs/contribute/style/style-guide.md

```diff
@@ -0,0 +1,1207 @@
+{{< /table >}}
+-->
+### 尽量使用通用描述而不是组件名称
+
+{{< table caption = "组件名称与通用描述" >}}
+可以 | 不可以
+:--| :-----
+Kubernetes API 服务器提供 OpenAPI 规范。| apiserver 提供 OpenAPI 规范 
+聚合 APIs 是下级 API 服务器。 | 聚合 APIs 是下级 APIServers。
+{{< /table >}}
+
+<!--
+### Use normal style for string and integer field values
+
+For field values of type string or integer, use normal style without quotation marks.
+
+{{< table caption = "Do and Don't - Use normal style for string and integer field values" >}}
+Do | Don't
+:--| :-----
+Set the value of `imagePullPolicy` to Always. | Set the value of `imagePullPolicy` to "Always".
+Set the value of `image` to nginx:1.16. | Set the value of `image` to `nginx:1.16`.
+Set the value of the `replicas` field to 2. | Set the value of the `replicas` field to `2`.
+{{< /table >}}
+-->
+### 使用普通样式表达字符串和整数字段值
+
+对于字符串或整数，使用正常样式，不要带引号。
+
+{{< table caption = "字符串和整数字段值约定" >}}
+可以 | 不可以
+:--| :-----
+将 `imagePullPolicy` 设置为 Always。 | 将 `imagePullPolicy` 设置为 "Always"。
+将 `image` 设置为 nginx:1.16. | 将 `image` 设置为 `nginx:1.16`。
+将 `replicas` 字段值设置为 to 2. | 将 `replicas` 字段值设置为 `2`.
+{{< /table >}}
+
+<!--
+## Code snippet formatting
+
+### Don't include the command prompt
+
+{{< table caption = "Do and Don't - Don't include the command prompt" >}}
+Do | Don't
+:--| :-----
+kubectl get pods | $ kubectl get pods
+{{< /table >}}
+-->
+## 代码段格式
+
+### 不要包含命令行提示符
+
+{{< table caption = "命令行提示符约定" >}}
+可以 | 不可以
+:--| :-----
+kubectl get pods | $ kubectl get pods
+{{< /table >}}
+
+<!--
+### Separate commands from output
+
+Verify that the Pod is running on your chosen node:
+
+    kubectl get pods --output=wide
+
+The output is similar to this:
+
+    NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
+    nginx    1/1       Running   0          13s    10.200.0.4   worker0
+-->
+### 将命令和输出分开
+
+例如：
+
+验证 Pod 已经在你所选的节点上运行：
+
+    kubectl get pods --output=wide
+
+输出类似于：
+
+    NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
+    nginx    1/1       Running   0          13s    10.200.0.4   worker0
+
+<!--
+### Versioning Kubernetes examples
+
+Code examples and configuration examples that include version information should be consistent with the accompanying text.
+
+If the information is version specific, the Kubernetes version needs to be defined in the `prerequisites` section of the [Task template](/docs/contribute/style/page-content-types/#task) or the [Tutorial template](/docs/contribute/style/page-content-types/#tutorial). Once the page is saved, the `prerequisites` section is shown as **Before you begin**.
+
+To specify the Kubernetes version for a task or tutorial page, include `min-kubernetes-server-version` in the front matter of the page.
+-->
+### 为 Kubernetes 示例给出版本
+
+代码示例或者配置示例如果包含版本信息，应该与对应的文字描述一致。
+
+如果所给的信息是特定于具体版本的，需要在
+[任务模版](/docs/contribute/style/page-content-types/#task)
+或[教程模版](/docs/contribute/style/page-content-types/#tutorial)
```

Comment:
```suggestion
或 [教程模版](/docs/contribute/style/page-content-types/#tutorial)
```

Reply:
this space now can be handled properly by the renderer engine.